testsuite/features/support/constants.rb

```diff
@@ -395,6 +395,7 @@
     'SLE-Product-SLES15-SP4-Pool for x86_64' => 'sle-product-sles15-sp4-pool-x86_64',
     'SLE-Product-SLES15-SP5-Pool for x86_64' => 'sle-product-sles15-sp5-pool-x86_64',
     'SLE-Product-SLES15-SP6-Pool for x86_64' => 'sle-product-sles15-sp6-pool-x86_64',
+    'SLE-Product-SLES15-SP7-Pool for x86_64' => 'sle-product-sles15-sp7-pool-x86_64',
```

Comment:
that would rather be for the "SP7" card but fine

Reply:
no I think this is used for label for the channel to be used in the activation key, either way at some point we need it